CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Breaking changes
+You must make the following changes when you migrate to this release, or your service may break.
+
+#### Deprecated: $govuk-border-width-form-element-error
```

Comment:
Only a small thing but the wording of the heading suggests that we're only now deprecating the variable even though we did it in https://github.com/alphagov/govuk-frontend/pull/1870. So maybe 'Removed' instead of 'Deprecated'?